docs/config.md

```diff
@@ -45,6 +45,7 @@ experimental   = true
 | `hosts_dir`         | `--hosts-dir`                      |                           | `certs.d` directory                                                                                                                                              | Since 0.16.0     |
 | `experimental`      | `--experimental`                   | `NERDCTL_EXPERIMENTAL`    | Enable  [experimental features](experimental.md)                                                                                                                 | Since 0.22.3     |
 | `host_gateway_ip`   | `--host-gateway-ip`                | `NERDCTL_HOST_GATEWAY_IP` | IP address that the special 'host-gateway' string in --add-host resolves to. Defaults to the IP address of the host. It has no effect without setting --add-host | Since 1.3.0      |
+| `bridge_ip`         | `--bridge-ip`                       | `NERDCTL_BRIDGE_IP`      | IP address for the default nerdctl bridge network              | Since 1.7.8 |
```

Comment:
```suggestion
| `bridge_subnet`         | `--bridge-subnet`                       | `NERDCTL_BRIDGE_SUBNET`      | IP subnet for the default nerdctl bridge network, e.g., 10.1.100.0/24             | Since 2.1.0 |
```

Reply:
The current implementation actually follows docker convention of `bip` which requires a specific IP and a CIDR. Any reason to prefer  subnet instead of IP? I can make the change to use subnet instead and let nerdctl choose the first IP of the subnet as gateway.

Reply:
I thought specifying CIDR might be more consistent with CNI, but it seems also fine to follow `dockerd --bip=<IP>` convention.

No strong opinion from me.